pkgs/applications/science/engineering/strictdoc/default.nix

```
@@ -0,0 +1,117 @@
+{ lib
+, stdenv
+, buildPythonApplication
+, fetchFromGitHub
+, python3
+, pythonOlder
+, black
+, flake8
+, guzzle_sphinx_theme
+, html5lib
+, invoke
+, mypy
+, openpyxl
+, poetry-core
+, pylint
+, pytest
+, sphinx
+, tidylib
+, beautifulsoup4
+, dataclasses
+, datauri
+, docutils
+, jinja2
+, lxml
+, markupsafe
+, pygments
+, reqif
+, setuptools
+, textx
+, xlrd
+, XlsxWriter
+}:
+
+buildPythonApplication rec {
+  pname = "strictdoc";
+  version = "0.0.25";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "${pname}-project";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-idNhF2I1Z0gjT35vDq1ELSZq69eg9JKf8jgQpSr6UHE=";
+  };
+
+  nativeBuildInputs = [
+    black
+    flake8
+    guzzle_sphinx_theme
+    html5lib
+    invoke
+    mypy
+    openpyxl
+    poetry-core
+    pylint
+    pytest
+    sphinx
+    tidylib
+    # filecheck
+    # lit
+  ];
+
+  propagatedBuildInputs = [
+    beautifulsoup4
+    datauri
+    docutils
+    jinja2
+    lxml
+    markupsafe
+    pygments
+    reqif
+    setuptools # ModuleNotFoundError: No module named 'pkg_resources'
```

```suggestion
    setuptools
```
That should be common knowledge for people working in pythonPackages.